storage/innobase/srv/srv0start.cc

```diff
@@ -1292,14 +1292,10 @@ dberr_t srv_start(bool create_new_db)
 		return(srv_init_abort(DB_ERROR));
 	}
 
-#ifdef LINUX_NATIVE_AIO
+#if defined(LINUX_NATIVE_AIO) || defined(HAVE_URING)
```

Comment:
I think we can live without those ifdefs, then we'll get some output for non-Linux OSes.